Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -3551,6 +3551,16 @@ static bool allowMinMaxPercentagesInAutoHeightBlocksQuirk()
 #endif
 }
 
+bool RenderBox::shouldComputePreferredLogicalWidthsFromStyle() const
+{
+    if (isRenderBlock()) {
```

Comment:
what makes "should compute from style" renderer type specific?

Reply:
After taking another look I think maybe it should not be... I can' quite tell what's going on with the `!isRenderTableCell()` bit in `RenderBlock` though...

Reply:
Actually yeah it probably doesn't make sense for `shouldComputePreferredLogicalWidthsFromStyle` either so I removed that specific renderer check too